webapp/src/pages/Mobility.jsx

```diff
@@ -12,6 +13,17 @@ export default function Mobility() {
       <LayoutContainer top gutters>
         <h5>{t("mobility:title")}</h5>
         <p className="text-secondary">{t("mobility:intro")}</p>
+        <hr />
+        {mdx("mobility:ride_connection_notice", {
```

Comment:
This should be put into `mobility:intro`. There is a difference between putting stuff like this in localization data (which is just a file that can be swapped out) and in the actual code (which cannot vary or be modified externally).